beta/src/sidebarReference.json

```diff
@@ -230,6 +230,66 @@
               "wip": true
             }
           ]
+        },
+        {
+          "title": "ReactDOM Attributes",
+          "path": "/apis/dom-attributes/domattributes",
+          "routes": [
+            {
+              "title": "aria",
+              "path": "/apis/dom-attributes/aria"
+            },
+            {
+              "title": "checked",
+              "path": "/apis/dom-attributes/checked"
+            },
+            {
+              "title": "className",
+              "path": "/apis/dom-attributes/classname"
```

Comment:
After my PR, I think these URLs and filenames should be camelCase, so this would be `className`.

Reply:
Cool, will change.